src/main/java/core/basesyntax/color/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.color;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);

```

Comment:
I would recommend here to make `Random` instance a class member. In such way we will avoid creating instance each time method is invoked.